### ISSUE TITLE
Path collision when using multilevel nav imports

### ISSUE THREAD

**Issue by ramonbroox** [jdoiro3/mkdocs-multirepo-plugin#29]
When using nav based imports, if two subsections have the same **title**, mkdocs crashes due to multirepo creating the same path for both:

For example:
```yml
nav:
  - Product Manual:
    - My Product: '!import aaa...' 
  - Developer Manual:
    - My Product: '!import bbb..'
```

Causes a crash due to `OSError: [Errno 66] Directory not empty`
(Also, the URLs only use the last navigation item, so you can't distinctly link `Product%20Manual/My%20Product` vs. `Developer%20Manual/My%20Product`).

**Comment by jdoiro3** [comment]
@ramonbroox, this should be fixed in `0.4.2`. Let me know if you're having issues still.

**Comment by jdoiro3** [comment]
There was a bug in 0.4.2 so please upgrade to 0.4.3.
